communication-et-langages.csl

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" default-locale="fr-FR" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Communication et Langages</title>
+    <id>http://www.zotero.org/styles/communication-et-langages</id>
+    <link href="http://www.zotero.org/styles/communication-et-langages" rel="self"/>
+    <link href="http://www.zotero.org/styles/traces" rel="template"/>
+    <link href="http://www.zotero.org/styles/american-political-science-association" rel="template"/>
+    <link href="http://www.necplus.eu/action/displayJournal?jid=CML" rel="documentation"/>
+    <author>
+      <name>Pierre-Carl Langlais</name>
+      <email>pierrecarl.langlais@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="humanities"/>
+    <issn>0336-1500</issn>
+    <eissn>1778-7459</eissn>
+    <summary>Communication et Langages style.</summary>
+    <updated>2015-11-16T22:02:33+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor">
+    <names variable="editor" delimiter=" ">
+      <name form="long" and="text" delimiter-precedes-last="never" sort-separator=" " font-style="normal"/>
+    </names>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name and="text" sort-separator=" " delimiter=", " delimiter-precedes-last="never"/>
+      <label form="short" prefix=" " text-case="lowercase"/>
+      <substitute>
+        <names variable="editor"/>
```

Comment:
you almost certainly want the substitute lines in `author` and `author-short` to be the same, so either add translator here or remove it below in author-short.